src/styles/primeng-theme/components/_splitbutton.scss

```diff
@@ -30,4 +30,49 @@ p-splitbutton {
             padding-right: 0;
         }
     }
+}
+
+p-splitbutton[iconButtonOptions] {
```

Comment:
The "icon" attribute already exist on the split button component and I can't rewrite it. I rename the class to "iconButton"